### PR TITLE
feat: [CDS-74527]: updated UI based on PM/UX Feedback

### PR DESCRIPTION
**Pull request by sonaligoyal-harness**
### Summary

- Update event correlation search api for trigger explorer page
- Add link of specific trigger  in explorer page to link it to activity history page
- Show time from eventCreatedAt in activity page
- Show  entire message for status on hover of status in activity page

#### Screenshots

Trigger activity history page:

Screenshot 2023-07-25 at 2 00 52 PM

Trigger explorer page linking to activity page:

https://github.com/harness/harness-core-ui/assets/98508399/300b092b-852a-4886-b060-aa6896495cfa

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/72-triggers/pages/triggers/TriggerLandingPage/TriggerActivityHistoryPage/TriggerActivityList.tsx [hunk 1]
```diff
@@ -114,12 +143,12 @@ const TriggerActivityList: React.FC<TriggerActivityListProps> = ({ triggersListR
...
         Header: getString('triggers.activityHistory.executionDetails'),
         accessor: row => row?.targetExecutionSummary?.planExecutionId,
-        width: '25%',
+        width: '30%',
         Cell: RenderColumnExecutionId
       },
       {
         Header: getString('common.payload'),
-        width: '10%',
+        width: '5%',
```
Comment (ishantmehtaharness) [inline src/modules/72-triggers/pages/triggers/TriggerLandingPage/TriggerActivityHistoryPage/TriggerActivityList.tsx:151]: Did we test this for smaller screen sizes?

File: src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx [hunk 2]
```diff
@@ -8,34 +8,74 @@
...
+    <Layout.Horizontal>
+      <Link
+        to={routes.toTriggersActivityHistoryPage({
+          accountId,
+          orgIdentifier: defaultTo(row.original.orgIdentifier, ''),
+          projectIdentifier: defaultTo(row.original.projectIdentifier, ''),
+          pipelineIdentifier: defaultTo(row.original.targetIdentifier, '-1'),
+          triggerIdentifier: defaultTo(row.original.triggerIdentifier, '')
+        })}
+        target="_blank"
```
Comment (ishantmehtaharness) [inline src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx:64]: Do we need `target="_blank"` while using `Link` component?

**Review by ishantmehtaharness — DISMISSED** [review]
2 comments, rest looks good

File: src/modules/72-triggers/pages/triggers/TriggerLandingPage/TriggerActivityHistoryPage/TriggerActivityList.tsx (hunk 1 above)
Comment (sonaligoyal-harness) [reply src/modules/72-triggers/pages/triggers/TriggerLandingPage/TriggerActivityHistoryPage/TriggerActivityList.tsx:151]: Yes, it works fine.

File: src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx (hunk 2 above)
Comment (sonaligoyal-harness) [reply src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx:64]: Yes, to redirect it to new tab.

File: src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx [hunk 3]
```diff
@@ -8,34 +8,74 @@
...
+}
+
+const RenderTriggerName: CellType = ({ row }) => {
+  const { accountId } = useParams<AccountPathProps>()
+  return (
+    <Layout.Horizontal>
+      <Link
+        to={routes.toTriggersActivityHistoryPage({
+          accountId,
+          orgIdentifier: defaultTo(row.original.orgIdentifier, ''),
```
Comment (pankajsingh-harness) [inline src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx:59]: All these values can be used via the spread. 
```
const {orgIdentifier = '', projectIdentifier='', targetIdentifier='', triggerIdentifier=''} = row.original
```

File: src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx (hunk 2 above)
Comment (ishantmehtaharness) [reply src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx:64]: I see, got it.

File: src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx [hunk 4]
```diff
@@ -8,34 +8,75 @@
...
+        })}
+        target="_blank"
+      >
+        <Text
+          font={{ variation: FontVariation.LEAD }}
+          color={Color.PRIMARY_7}
+          tooltipProps={{ isDark: true }}
+          lineClamp={1}
+        >
+          {row.original.triggerIdentifier}
```
Comment (pankajsingh-harness) [inline src/modules/72-triggers/pages/triggers/views/TriggerExplorer.tsx:73]: NIT: 
```suggestion
          {triggerIdentifier}
```

**Review by pankajsingh-harness — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**